PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -784,6 +1289,64 @@ HTREEITEM ProjectPanel::addFolder(HTREEITEM hTreeItem, const TCHAR *folderName)
 	return addedItem;
 }
 
+HTREEITEM ProjectPanel::addDirectory(HTREEITEM hTreeItem, const TCHAR *folderName, bool root, const TCHAR *monitorPath, const std::vector<generic_string>* filters)

```

Comment:
Ok that's enough @ariccio. I can not let you say this kind of things. We are not hostile to code cleanup and *certainly* not to refactoring since it is the current direction. It is because of the current refactoring that we can not accept all PR. I know, it takes time, but we do what we can. I think that you would already have understood that. You disagree with our methods and I understand that and maybe we're wrong. However, I don't know what you're looking for, but please calm down a bit...